api/src/main/java/org/apache/iceberg/expressions/ExpressionUtil.java

```diff
@@ -515,6 +523,8 @@ private static String sanitize(Type type, Object value, long now, int today) {
         return "(time)";
       case TIMESTAMP:
         return sanitizeTimestamp((long) value, now);
+      case TIMESTAMP_NANO:
```

Comment:
I think I'd like to double check if sanitizing the `TIMESTAMP_NANO` as microseconds is OK?
Looking at the test cases it seems fine for addressing the existing cases but those are all coarser grained time differences (like 90 minutes, or "about-now" within 5 minutes). But are there cases where the sanitized string need to represent something more granular for nanosecond timestamps?

Reply:
I think it's good, I stepped through, really sanitization should just provide a useful description of the data values, and (timestamp) and the other cases are sufficient.

Reply:
I think it is fine to sanitize as micros. The time ranges for sanitization are very coarse.